doc/WebSite/Entity-Framework-Core.md

```diff
@@ -262,3 +262,13 @@ to use the TaskRepository.
     layer** (.Core project in the startup template). Then define the custom repository
     **classes** in the **.EntityFrameworkCore** project if you want to
     abstract EF Core from your domain/application.
+
+### Other Database Integrations
+
+This document is prepared via the MS SQL Server example. 
```

Comment:
This document and the examples is based on using MS SQL Server.

Reply:
"`are` based on"